src/store/modules/getters/wallet.js

```diff
@@ -3,3 +3,4 @@ export const getSigner = state => state.signer;
 export const getConnector = state => state.connector;
 export const getLikerInfo = state => state.likerInfo;
 export const walletMethodType = state => state.methodType;
+export const walletLIKEBalance = state => Number(state.likeBalance).toFixed(2);
```

Comment:
Why `toFixed` here?  Should handle rounding & formatting at the very end with `formatNumber()`